daq/report.py

```diff
@@ -111,9 +123,22 @@ def finalize(self):
         self._writeln(self._TEST_SEPARATOR % self._REPORT_COMPLETE)
         self._file.close()
         self._file = None
-        if self._alt_path:
+        self._write_pdf_report()
```

Comment:
extract the ~7 ilnes above this (starting with self._file) to a function _write_md_report()